src/api/loader.coffee

```diff
@@ -58,10 +58,12 @@ handleAPIEvent = (apiEventChannel, baseUrl, setting, requestEvent = {}) ->
     $.ajax(apiSetting)
       .then((responseData) ->
         delete LOADING[apiSetting.url]
-        completedEvent = interpolate(setting.completedEvent, requestEvent.data)
-        completedData = getResponseDataByEnv(isLocal, requestEvent, responseData)
-        apiEventChannel.emit(completedEvent, completedData)
-
+        try
```

Comment:
Could this be an `if response? ... else apiEventChannel.emit('error', ...)` instead of `try/catch`?